src/themes/light.scss

```diff
@@ -3,17 +3,29 @@
   --wxc-text-color: #{$wxc-black-alpha-100};
   --wxc-meeting-border: 1px solid #{$wxc-black-alpha-20};
   --wxc-meeting-box-shadow: 0px 24px 48px #{$wxc-black-alpha-20}, 0px 0px 1px #{$wxc-black-alpha-12};
-  
+
+  --wxc-banner-background: #{$wxc-gradation-light-transparent};
+  --wxc-banner-text-color: rgba(0, 0, 0, 0.95);
+
+  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner.my-preview { display: flex; }
+
+  // Icons
   --wxc-icon--color: #{$wxc-black-alpha-100};
   --wxc-icon--active-color: #{$wxc-green-50};
   --wxc-icon--muted-color: #{$wxc-red-60};
+  --wxc-icon--audio-microphone--active-color: #{$wxc-green-50};
+  --wxc-icon--audio-microphone--color: #{$wxc-black-alpha-100};
+  --wxc-icon--audio-microphone-muted--color: #{$wxc-red-60};
+  --wxc-icon--camera--active-color: #{$wxc-green-50};
+  --wxc-icon--camera-muted--color: #{$wxc-red-60};
+  --wxc-icon--camera--color: #{$wxc-black-alpha-100};
+  --wxc-icon--settings--color: #{$wxc-black-alpha-100};
+  --wxc-icon--share-screen--color: #{$wxc-black-alpha-100};
+  --wxc-icon--cancel--color: #{$wxc-black-alpha-100};
+  --wxc-icon--participant-list--color: #{$wxc-black-alpha-100};
 
   .wxc-old-icon { display: none; }
   .wxc-icon { display: block; }
-}
 
-.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-light {
-  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster .title {
-    display: block;
-  }
+  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster .title { display: block; }
```

Comment:
it's ok to leave like it was, a separate block for these temporary logic properties:
```css
.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-light {
  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster .title { display: block; }
}